polkadot-parachains/integritee-runtime/src/xcm_config.rs

```diff
@@ -126,10 +134,12 @@ impl Convert<MultiLocation, Option<CurrencyId>> for CurrencyIdConvert {
 				X2(Parachain(para_id), junction)
 					if junction == TEER_GENERAL_KEY && para_id == self_para_id =>
 					Some(CurrencyId::TEER),
+				Here => Some(CurrencyId::RelayNative),
 				_ => None,
 			},
 			MultiLocation { parents, interior } if parents == 0 => match interior {
 				X1(junction) if junction == TEER_GENERAL_KEY => Some(CurrencyId::TEER),
+				Here => Some(CurrencyId::TEER),
```

Comment:
I think we'll need that because that will be how Asset Hub refers to TEER (different from Karura, Moonriver style)